src/chat/Chat.js

```diff
@@ -51,11 +57,18 @@ export default class Chat extends PureComponent<Props> {
 
   render() {
     const { styles } = this.context;
+    console.log('styles', styles.screen);
     const { isFetching, narrow, isOnline, noMessages } = this.props;
     const showMessagePlaceholders = noMessages && isFetching;
+    const offset = Platform.OS === 'ios' ? 0 : -400;
```

Comment:
Hardcoding this will not give us the correct value most of the time.
The size varies depending on if the layout is Portrait or Landscape, if the autocorrect is enabled and possibly other settings (emoji keyboard, other language keyboard, custom etc).